.github/ISSUE_TEMPLATE/4-REGULAR_RELEASE_CHECKLIST.md

```diff
@@ -3,14 +3,19 @@ name: "\U0001F680 Regular Release Checklist"
 about: "\U0001F512 Maintainers only: create a checklist for a regular release process"
 title: 'Release checklist for v2.x.x'
 labels: 'i: scope:distribution'
-assignees: 'schlessera'
+assignees: '@wp-cli/maintainers'
```

Comment:
I don't think this works, actually:

<img width="347" alt="CleanShot 2023-11-01 at 15 38 48@2x" src="https://github.com/wp-cli/wp-cli/assets/36432/0dce19db-f647-47f2-ab74-293b50545ea4">

We could probably leave it empty.

Reply:
Hmm only works for reviewers then I guess